molecule/default/molecule.yml

```diff
@@ -23,7 +23,3 @@ provisioner:
   name: ansible
   playbooks:
     converge: converge.yml
-scenario:
-  name: default
-verifier:
-  name: testinfra
```

Comment:
Why this removal?

Reply:
testinfra is no longer the default verifier in molecule 3.0, ansible is (which we're using here)
https://molecule.readthedocs.io/en/latest/configuration.html#verifier